keras-sprint/deeplabv3_plus.ipynb

```diff
@@ -0,0 +1,4573 @@
+{
```

Comment:
<p>I feel this example is particularly challenging, the readers' attention will then be split and instead of focusing in the push part they will focus too much in the model part. Maybe consider a simpler example, such as just using a simple mnist CNN.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/community-events/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
I added a disclaimer on top. I will do a simple RNN one to demonstrate text example, I'd keep this one, I did it because it's Gradio UI is good.